packages/svelte2tsx/src/nodes/event-handler.ts

```diff
@@ -47,17 +45,9 @@ function getEventDefExpressionForNonCompoent(eventName: string, ele: Node) {
 }
 
 export function eventMapToString(events: Map<string, string | string[]>) {
-    return '{' +
-        Array.from(events.entries()).map(eventMapEntryToString).join(', ') +
-        '}';
-
+    return '{' + Array.from(events.entries()).map(eventMapEntryToString).join(', ') + '}';
 }
 
-function eventMapEntryToString([eventName, expression]: [
-    string,
-    string | string[]
-]) {
-    return `'${eventName}':${
-        Array.isArray(expression) ? `[${expression}]` : expression
-        }`;
+function eventMapEntryToString([eventName, expression]: [string, string | string[]]) {
```

Comment:
I think we can add back the `SvelteExtractEvent` type alias because of this is still possibly return array

Reply:
Good catch! I solved it in another way by adding a `unionType` method.